keps/sig-network/2593-multiple-cluster-cidrs/README.md

```diff
@@ -827,14 +935,18 @@ checking if there are objects with field X set) may be a last resort. Avoid
 logs or events for this purpose.
 -->
 
+`multicidrset_cidrs_allocations_total` metric with value > 0 will indicate enablement. 
+The operator can also check if ClusterCIDR objects are created, this can be done
+using `kubectl get cc -A`
+
 ###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
 
 We will carry-over existing metrics to the new controller: https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/nodeipam/ipam/cidrset/metrics.go#L26-L68
 
 They are:
--   cidrset_cidrs_allocations_total - Count of total number of CIDR allcoations
--   cidrset_cidrs_releases_total - Count of total number of CIDR releases
--   cidrset_usage_cidrs - Gauge messuring the percentage of the provided CIDRs
+-   multicidrset_cidrs_allocations_total - Count of total number of CIDR allcoations
+-   multicidrset_cidrs_releases_total - Count of total number of CIDR releases
+-   multicidrset_usage_cidrs - Gauge messuring the percentage of the provided CIDRs
     that have been allocated
 
```

Comment:
The health of the service can be determined if the *_total metrics increase with the number of nodes and the `usage_cidrs` metric is greater or equal than the number of nodes

It is important to mention that the Service is optional, it depends on the CNI implemented in the cluster, you can have the multicidr allocator and be a noOp in the cluster, if the CNI doesn't make use of the information provided

Reply:
Done

I added that it is optional and based on `--allocate-node-cidrs` being set to true in kube-controller-manager

I am not sure about the CNI specific behavior.

I believe when  `--allocate-node-cidrs` is set to true, this controller will still add the PodCIDRs to the Node object. However the CNI can ignore if it manages the ipam?

Reply:
yes,, the field is optional to consume, it doesn't imply the cluster will use it